### DISCUSSION TITLE
CRUD

### DISCUSSION THREAD

**Discussion by lovesL** [tolking/element-pro-components#489] [Q&A] [answered]
![image](https://github.com/tolking/element-pro-components/assets/27960630/1b413005-d8b7-4a49-8d5e-ae9af78a5ab2)
这个rule能拆分searchRule  和formRule嘛 我知道可以用defineFormColumns重新定义form 但是真是太麻烦了 又要吧searchForm的的字段全部copy过来一个一个加component 加rule 。 如果不是对form布局有特殊要求其实根本用不上defineFormColumns 希望能把rule拆分一下

**Comment by tolking** [comment, marked as answer]
拆分会带来额外的心里负担。

可以使用 ProCrud 中的 search-rules 与 rules 属性

**Reply by lovesL** [reply]
好的

**Comment by lovesL** [comment]
已解决
